tensorflow/contrib/tpu/profiler/pip_package/setup.py

```diff
@@ -52,12 +52,24 @@
         #   4 - Beta
         #   5 - Production/Stable
         'Development Status :: 3 - Alpha',
-
-        # Indicate who your project is intended for
+        
         'Intended Audience :: Developers',
-        'Topic :: Software Development',
+        'Intended Audience :: Education',
+        'Intended Audience :: Science/Research',
+        
         'License :: OSI Approved :: Apache Software License',
+        
+        'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```

Comment:
Do we also need to list other python versions we support?
we also support python 3.4 and 3.5